rocketc/static/js/src/studio_view.js

```diff
@@ -47,4 +47,22 @@ function StudioViewEdit(runtime, element) {
         $("#select-default").attr("class", "button");
     });
 
+    $("#xb-field-edit-channel").on("change", function() {
+
+        if(this.value === "Specific Channel"){
```

Comment:
Here we have a problem using hardcoded values. What happen if you change the string "Specific Channel" in the xlobck class? you'll never enter in this if block. Is there any way to pass values from your xblock class to js files?

Reply:
It is possible create a method to read that